server/graphql/v2/query/collection/ContributorsCollectionQuery.ts

```diff
@@ -0,0 +1,153 @@
+import assert from 'assert';
+
+import { GraphQLBoolean, GraphQLInt, GraphQLList, GraphQLNonNull } from 'graphql';
+import { isNil } from 'lodash';
+
+import { ifStr } from '../../../../lib/utils';
+import { sequelize } from '../../../../models';
+import { BadRequest } from '../../../errors';
+import { GraphQLContributorCollection } from '../../collection/ContributorCollection';
+import { AccountTypeToModelMapping, GraphQLAccountType } from '../../enum/AccountType';
+import { GraphQLMemberRole } from '../../enum/MemberRole';
+import { fetchAccountWithReference, GraphQLAccountReferenceInput } from '../../input/AccountReferenceInput';
+import { GraphQLChronologicalOrderInput } from '../../input/ChronologicalOrderInput';
+import EmailAddress from '../../scalar/EmailAddress';
+
+const ContributorsCollectionQuery = {
+  description: 'Get Contributors grouped by their profiles',
+  type: new GraphQLNonNull(GraphQLContributorCollection),
+  args: {
+    account: {
+      type: GraphQLAccountReferenceInput,
+      description: 'Host hosting the account',
+    },
+    host: {
+      type: GraphQLAccountReferenceInput,
+      description: 'Host hosting the account',
+    },
+    limit: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 100 },
+    offset: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 0 },
+    role: { type: new GraphQLList(GraphQLMemberRole) },
+    type: { type: new GraphQLList(GraphQLAccountType) },
+    email: {
+      type: EmailAddress,
+      description: 'Admin only. To filter on the email address of a member, useful to check if a member exists.',
+    },
+    orderBy: {
+      type: new GraphQLNonNull(GraphQLChronologicalOrderInput),
+      defaultValue: { field: 'createdAt', direction: 'ASC' },
+      description: 'Order of the results',
+    },
+    includeInherited: {
+      type: GraphQLBoolean,
+      defaultValue: true,
+    },
```

Comment:
Maybe add a description for this one, it's not obvious. And apparently not used?